### PR TITLE
316405_revamp sample for datasource iframe js embedding

### PR DESCRIPTION
**Pull request by dharshinisf21**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
+
+  This application aims to demonstrate how to embed the data source designer module into your JavaScript-based web application using an iFrame.
+  
+# Datasource View
+
+ ![Datasource_Iframe_js view](https://github.com/boldbi/iframe-datasource-javascript-sample/assets/129486688/93584224-edee-4482-b569-b4e6cd81de4f)
+
+## Prerequisites
+
+ * [Node.js](https://nodejs.org/en/)
```
Comment (vinothkumar1732) [inline README.md:11]: Change the Node.js as NodeJS

File: README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
...
+ * [Visual Studio Code](https://code.visualstudio.com/download)
+
+ > **NOTE:** Node.js v14.16 to v18.16 are supported.
```
Comment (vinothkumar1732) [inline README.md:14]: Change the Node.js as NodeJS

File: DatasourceListing.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+         </div>
+         <div id ="panel"></div>    
+      </div>
+      <!--Datasource view page for create and edit-->
+      <div id="viewer-section">
+         <div id="viewer-header">
+               <button id="create-datasource">Create</button>
+         </div>
+         <div id="datasource">
+            <div id="message" style="display: block; text-align: center; margin-top: 235px; font-size: 20px;">Please select any Data source to Edit</div>
```
Comment (vinothkumar1732) [inline DatasourceListing.html]: To edit, please select any Data Source.

**Review by vinothkumar1732 — CHANGES_REQUESTED** [review]

File: DatasourceListing.html (hunk 3 above)
Comment (dharshinisf21) [reply DatasourceListing.html]: Added this line

File: README.md [hunk 4]
```diff
@@ -1,2 +1,74 @@
-# iframe-datasource-javascript-sample
-This application aims to demonstrate how to embed the data source designer module into your JavaScript-based web application using an iFrame.
+# BoldBI Datasource Iframe with Javascript Embedding
```
Comment (vinothkumar1732) [inline README.md]: Data Source Iframe Embedding with JavaScript

File: README.md [hunk 5]
```diff
@@ -1,2 +1,74 @@
...
+
+  This application aims to demonstrate how to embed the data source designer module into your JavaScript-based web application using an iFrame.
+  
+# Datasource View
```
Comment (vinothkumar1732) [inline README.md]: Data Source view

File: README.md [hunk 6]
```diff
@@ -1,2 +1,74 @@
...
+
+* Copy the downloaded `embedConfig.json` file and paste it into the designated [location](https://github.com/boldbi/iframe-datasource-javascript-sample/tree/master) within the application. Please ensure that you have placed it in the application as shown in the following image.
+  
+    ![EmbedConfig image](https://github.com/bold-bi/embedded-bi-samples/assets/129486688/e7c59dc4-4dfe-4495-a396-b7c21fafc6ff)
+
+ ## Developer IDE
+
+  * Visual studio code(https://code.visualstudio.com/download)
+  
+### Run a Sample Using Visual Studio Code
```
Comment (vinothkumar1732) [inline README.md]: Run a Sample using Visual Studio Code

**Review by vinothkumar1732 — APPROVED** [review]
Do the minor changes and other content are fine.

File: README.md (hunk 4 above)
Comment (dharshinisf21) [reply README.md]: Updated this line

File: README.md (hunk 5 above)
Comment (dharshinisf21) [reply README.md]: Updated this line

File: README.md (hunk 6 above)
Comment (dharshinisf21) [reply README.md]: Updated this line

**Pull request merged**